docs/docs/ref-01-top-level-api.md

```diff
@@ -89,9 +89,13 @@ If the optional callback is provided, it will be executed after the component is
 
 > Note:
 >
-> `React.render()` replaces the contents of the container node you
-> pass in. In the future, it may be possible to insert a component to an
-> existing DOM node without overwriting the existing children.
+> `React.render()` controls the contents of the container node you pass in*. Any existing DOM elements
```

Comment:
The star is there to indicate a reference to the footnote/clarification below.  It's a way of connecting the additional information to that sentence without breaking the train of thought there.